lib/connectors/memory.js

```diff
@@ -502,7 +502,41 @@ Memory.prototype._findAllSkippingIncludes = function(model, filter) {
     // limit/skip
     const skip = filter.skip || filter.offset || 0;
     const limit = filter.limit || nodes.length;
+    // groupBy
     nodes = nodes.slice(skip, skip + limit);
+    if (filter.groupBy) {
```

Comment:
this will require support for connector too

Reply:
Yes. And it's implemented [here](https://github.com/loopbackio/loopback-connector/pull/466).